common/app/routes/Challenges/views/project/Project.jsx

```diff
@@ -45,7 +45,8 @@ export class Project extends PureComponent {
       isCompleted,
       description
     } = this.props;
-    const imageURL = '//i.imgur.com/' + image + '.png';
+    const imgurId = image ? image : 'ovKSXMs';
+    const imageURL = '//i.imgur.com/' + imgurId + '.png';
```

Comment:
We can make this simpler by using default expressions. Then the other changes become unnecessary.
```js
    const {
      id,
      title,
      image = 'KDJFLDK',
      isCompleted,
      description
    } = this.props;
```

Reply:
Will fix. Thanks!

Reply:
Updated to add the default image as outlined in the feedback.

Reply:
I went through the process outlined on the Contribute page for amending the PR and force pushed, but I am not seeing my latest updates show up on the PR.  I may have to review this again after work today.

Reply:
did you force push up?

Reply:
You may have missed adding the file too? Happens all the time 

```
git add <filename>
git commit --amend --no-edit
```

The force push to the branch this is already on.

Reply:
I believe I've done this now, but the build failed.  Is there an easy way to tell where to start troubleshooting the failed build?  I'm not really sure where to go next from here.